### PR TITLE
Self Assessment Navigation Bugfix

### PR DESCRIPTION
**Pull request by mattThousand**
### Why
We'd like to return to the root of the self assessment stack when pressing the `X` button and when pressing `Done` at the end of the self assessment.

### This commit
This commit navigates the user to the root of the self assessment stack upon pressing the `X` button during the course of the self assessment and also when the `done` button is pressed at the conclusion of the self assessment. This commit also hides the `X` button on the self assessment start screen, as it is redundant there.

Jun-12-2020 15-31-37

### CODE REVIEW

File: app/views/assessment/index.js [hunk 1]
```diff
@@ -87,8 +87,7 @@ const Assessment = ({ navigation }) => {
   const AssessmentCancel = () => (
     <TouchableOpacity
       onPress={() => {
-        navigation.goBack();
-        navigation.popToTop();
+        navigation.navigate('Start');
```
Comment (JacobJaffe) [inline app/views/assessment/index.js]: I realize this was already `Start`, but I'd recommend making route names more specific so there's no risk of duplicates in the tree.
e.g.
`AssessmentStart`

Comment (johnschoeman) [reply app/views/assessment/index.js]: perhaps `SelfAssessmentStart`

**Review by johnschoeman — APPROVED** [review]
thanks for grabbing this!

**Review by tparesi — APPROVED** [review]
LGTM - thanks!

**Review by JacobJaffe — APPROVED** [review]
🥳 Thanks for the naming changes

File: app/views/assessment/AssessmentStart.js [hunk 2]
```diff
@@ -18,12 +18,12 @@ import { Colors } from '../../styles';
...
 
 /** @type {React.FunctionComponent<{}>} */
-export const Start = ({ navigation }) => {
+export const AssessmentStart = ({ navigation }) => {
   let { t } = useTranslation();
   return (
     <Info
       ctaAction={() => {
-        navigation.push('Question', {
+        navigation.push('AssessmentQuestion', {
```
Comment (johnschoeman) [inline app/views/assessment/AssessmentStart.js:26]: should we use the Screens from navigation here?
```suggestion
        navigation.push(Screesn.AssessmentQuestion, {
```

File: app/views/assessment/index.js [hunk 3]
```diff
@@ -122,31 +120,32 @@ const Assessment = ({ navigation }) => {
...
               cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
               cardStyle: {
                 backgroundColor: 'transparent',
               },
             }}>
             <Stack.Screen
-              component={Start}
-              name='Start'
+              component={AssessmentStart}
+              name='AssessmentStart'
```
Comment (johnschoeman) [inline app/views/assessment/index.js:132]:
```suggestion
              name={Screens.AssessmentStart}
```

**Pull request merged**